design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -1,20 +1,16 @@
 type ComparableItem {
-    productId: ID! @doc(description: "Product Id")
-    name: String! @doc(description: "Product name")
-    sku: String! @doc(description: "Product SKU")
-    priceRange: ProductPriceRange! @doc(description: "Product prices")
-    canonical_url: String @doc(description: "Product URL")
-    images: [ProductImage]! @doc(description: "Product Images")
-    values: [ProductAttribute]! @doc(description: "Product comparable attributes")
+    uid: ID!
+    product: ProductInterface!
```

Comment:
After discussion with @cpartica and @melnikovi, decided to return ProductInterface here.